tests/source/arrow_in_comments/arrow_in_single_comment.rs

```diff
@@ -0,0 +1,10 @@
+// rustfmt-wrap_comments: true
```

Comment:
Just wondering, is `wrap_comments=true` needed to trigger the issue? I would expect the fix to work without needing to configure `wrap_comments`, but if it's necessary can you try to explain why?

Reply:
It has nothing to do with `wrap_comments` option. Mistakenly left after playing around some configs. :sweat_smile: